cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -551,16 +555,36 @@ def _add_pcluster_policies_to_role(self, role_ref: str, name: str):
                     iam.PolicyStatement(
                         sid="Ec2",
                         actions=[
-                            "ec2:DescribeVolumes",
-                            "ec2:AttachVolume",
                             "ec2:DescribeInstanceAttribute",
-                            "ec2:DescribeInstanceStatus",
-                            "ec2:DescribeInstances",
-                            "ec2:DescribeInstanceTypes",
                         ],
                         effect=iam.Effect.ALLOW,
                         resources=["*"],
                     ),
+                    iam.PolicyStatement(
+                        sid="S3GetObj",
+                        actions=["s3:GetObject"],
+                        effect=iam.Effect.ALLOW,
+                        resources=[
+                            self.format_arn(
+                                service="s3",
+                                resource="{0}-aws-parallelcluster/*".format(self.region),
+                                region="",
+                                account="",
+                            )
+                        ],
+                    ),
+                ]
+            ),
+            roles=[role_ref],
+        )
+
+    def _add_batch_head_node_policies_to_role(self, role_ref: str, name: str):
```

Comment:
Minor: could this method be moved to awsbatch_builder.py?

Reply:
the reason I didn't do this is because the head node is added prior to instantiating the batch construct and i wasn't sure if that was a problem. trying it now.

Reply:
this has been moved